Running "gmake test" was failing when it starts to try to do the three 
"test3" tests.  These tests depend upon files called:

   test3-data/01.input
   test3-data/01.answer
   test3-data/02.input
   test3-data/02.answer
   test3-data/03.input
   test3-data/03.answer

 These files are simply not included with the popt 1.7 source tarball.

 By googling around, they were found at:

 http://nesl.ee.ucla.edu/fw/han/old_machine_backup/overo-oe/tmp/work/i686-linux/popt-native-1.14-r2/popt-1.14/test3-data/

 but looking at the latest version of popt (1.16), and the testit.sh
 script in there has those three tests commented out:

 #run_diff test3 "test3 - 51" test3-data/01.input test3-data/01.answer
 #run_diff test3 "test3 - 52" test3-data/02.input test3-data/02.answer
 #run_diff test3 "test3 - 53" test3-data/03.input test3-data/03.answer

 This patch has been added to comment them out in a similar manner.

 If/when we upgrade to popt 1.16, then we can simply remove that patch. 

--- popt-1.7/testit.sh.orig	2012-12-14 07:49:41.381719596 -0800
+++ popt-1.7/testit.sh	2012-12-14 07:49:54.020453347 -0800
@@ -92,9 +92,9 @@
 run test1 "test1 - 39" "arg1: 0 arg2: (none) oStr: ping rest: pong" --optional=ping pong
 run test1 "test1 - 40" "arg1: 0 arg2: (none) oStr: ping rest: pong" --optional ping pong
 
-run_diff test3 "test3 - 41" test3-data/01.input test3-data/01.answer
-run_diff test3 "test3 - 42" test3-data/02.input test3-data/02.answer
-run_diff test3 "test3 - 43" test3-data/03.input test3-data/03.answer
+#run_diff test3 "test3 - 41" test3-data/01.input test3-data/01.answer
+#run_diff test3 "test3 - 42" test3-data/02.input test3-data/02.answer
+#run_diff test3 "test3 - 43" test3-data/03.input test3-data/03.answer
 
 echo ""
 echo "Passed."
